tests/apps/test_testenv.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+import pytest
+
+in_ci = os.environ.get("GITHUB_RUN_ID") is not None
+
+
+@pytest.mark.skipif(not in_ci, reason="only run in CI")
```

Comment:
Does having it a directory higher than the other tests ensure it gets run first?

Reply:
not sure, but i don't think order matters really, it was more that this isn't really app specific